tools/ete/ete_gene_cnv.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import print_function
+
+import optparse
+import collections
```

Comment:
Import statements are in the wrong order. 'import collections' should be before 'import optparse'

Reply:
Actually it would be good to use [argparse](https://docs.python.org/3/library/argparse.html), `optparse` has been deprecated for a while.